cli_tools/google-osconfig-agent/main.go

```diff
@@ -100,9 +100,7 @@ func main() {
 		logger.Errorf(err.Error())
 	}
 
-	if config.Debug() {
-		packages.DebugLogger = log.New(&logWritter{}, "", 0)
-	}
+	packages.DebugLogger = log.New(&logWritter{}, "", 0)
```

Comment:
We did, this just hooks up the logger, no logs will be written unless debug is set. This change is important so that debug can get updated with each setconfig